scripts/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# This script builds wheels for the API, SDK, and extension packages in the
+# dist/ dir, to be uploaded to PyPI.
+
+set -ev
+
+# Get the latest versions of packaging tools
+python3 -m pip install --upgrade pip setuptools wheel
+
+BASEDIR=$(dirname $(readlink -f $(dirname $0)))
+
+(
+  cd $BASEDIR
+  mkdir -p dist
+
+  # Clean old artifacts
+  rm -rf dist/*
+
+  # Clean old artifacts
+  for d in opentelemetry-api/ opentelemetry-sdk/ ext/*/ ; do
+    (
+      cd "$d"
+      echo "pwd"
+      echo $(pwd)
+      python3 setup.py bdist_wheel --dist-dir "$BASEDIR/dist/"
```

Comment:
I think we should add universal here, so that package will resolve on all platforms:

https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels

Or we could go modify everyone's setup.cfg, but that feels duplicative and error prone.

Reply:
Universal is about Python 2/3 compatibility only, it has AFAIK nothing to do with OS or architecture.